src/test/java/htsjdk/samtools/SAMUtilsTest.java

```diff
@@ -284,4 +286,46 @@ public void testCompressedBasesToBytes() {
                 'R', 'R', 'S', 'S', 'V', 'V', 'W', 'W', 'Y', 'Y', 'H', 'H', 'K', 'K', 'D', 'D', 'B', 'B'};
         Assert.assertEquals(new String(bytes), new String(expectedBases));
     }
+
+
+    @DataProvider()
+    public Iterator<Object[]> getOAValues(){
+        final SAMRecordSetBuilder builder = new SAMRecordSetBuilder();
+        final List<Object[]> tests = new ArrayList<>();
+
+        {
+            final SAMRecord record = builder.addFrag("test1", 0, 15, false, false, "36M", null, 45);
+
+            tests.add(new Object[]{record, "chr1,15,+,36M,255,"});
+        }
```

Comment:
Very weird, we don't often open scopes just to reuse variable names, but I don't think there's anything wrong with it here, and it's convenient to reuse the name record...  Maybe we should use this more when doing repetitive construction of things in a dataprovider...